index.js

```diff
@@ -14,7 +14,9 @@ const v6 = `
 (?:${v6seg}:){1}(?:(:${v6seg}){0,4}:${v4}|(:${v6seg}){1,6}|:)| // 1::              1::3:4:5:6:7:8   1::8            1::3:4:5:6:7:1.2.3.4
 (?::((?::${v6seg}){0,5}:${v4}|(?::${v6seg}){1,7}|:))           // ::2:3:4:5:6:7:8  ::2:3:4:5:6:7:8  ::8             ::1.2.3.4
 )(%[0-9a-zA-Z]{1,})?                                           // %eth0            %1
-`.replace(/\s*\/\/.*$/gm, '').replace(/\n/g, '').trim();
+`;
+const v6 = v6src.replace(/\s*\/\/.*$/gm, '').replace(/\n/g, '').trim();
+
```

Comment:
You'll need to rename `v6` on line 6 to `v6src`. I'm not sure why this is needed since the code shared in #14 should work even if the carriage returns are replaced with `\n`.

Reply:
You're right!

Reply:
please check now